setup.py

```diff
@@ -54,6 +54,7 @@ class _Ctx(object):
 
 INSTALL_REQUIRES = [
     'pyaaf2==1.4.0',
+    'future',
```

Comment:
Is this to enable the `builtins` module only in python2?  If so note that on line 59 there are some python2 specific dependencies, this could go there.

Reply:
I think it is needed for both python2 and python3.

Reply:
And sorry to not answer last week, I was away. Also I think it was simpler to do it the way we did it in the end :-)

Reply:
Looks like its in python3: https://docs.python.org/3/library/builtins.html

Reply:
Could you add a comment saying "# enables the builtins module on python2"